<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestParameterSettingClass</class>
 <widget class="QWidget" name="TestParameterSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestParameterSetting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>MaxBatchSize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MaxBatchSizeEdit">
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>BatchSize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="BatchSizeEdit">
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>maxContourCount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxContourCountEdit">
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>maxContourPointCount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxContourPointCountEdit">
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>maxInnerCountourCount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxInnerContourCountEdit">
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>DeviceType</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DeviceTypeComboBox">
         <property name="currentText">
          <string>E_GPU</string>
         </property>
         <item>
          <property name="text">
           <string>E_CPU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E_GPU</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>DeviceID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DeviceIDEdit">
         <property name="frame">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ComputePrecision</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ComputePrecisionComboBox">
       <property name="currentText">
        <string>E_FP32</string>
       </property>
       <item>
        <property name="text">
         <string>E_FP16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E_FP32</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
